learn/core_concepts/indexes.md

```diff
@@ -145,3 +145,15 @@ Typo tolerance is a built-in feature that helps you find relevant results even w
 You can update the typo tolerance settings using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update typo tolerance endpoint](/reference/api/settings.md#update-typo-tolerance-settings).
 
 [Learn more about typo tolerance.](/learn/configuration/typo_tolerance.md)
+
+## Swapping indexes
+
+Suppose you want to deploy a new version of your `movies` index in production. This `movies` index is where your users are currently making search requests. The changes you made are in `movies_new`. To sync `movies_new` with Meilisearch in production, you will use index swapping.
+
+This means that the schema, settings, and task history of  `movies` will be swapped with the schema, settings, and task history of `movies_new` **without any downtime for the search clients**. The task history of `enqueued` tasks is not modified.
+
+Once swapped, your users will still be making search requests to the `movies` index but it will contain the data of `movies_new`. You can delete `movies_new` after the swap or keep it in case something goes wrong and you want to swap back.
+
+Swapping indexes is an atomic transaction: **either all indexes are successfully swapped, or none are**.
+
+You can perform multiple swap operations using the [swap indexes endpoint](/reference/api/indexes.md#swap-indexes).
```

Comment:
The fact that you can perform multiple swap operations with a single API request is important but seems like a much better fit for the API reference than here. Since this is a core concepts document—not intended to be exhaustive—it's better to leave off practical details like this so that the key takeaways stand out more. Just my 2️⃣ 🪙 

```suggestion
For more information, see the [swap indexes endpoint](/reference/api/indexes.md#swap-indexes).
```